docs/policy-only.md

```diff
@@ -0,0 +1,170 @@
+# Running Antrea In Policy Only Mode
+
+Antrea supports chaining with routed CNI implementations such as EKS CNI. In this mode, Antrea
+enforces Kubernetes NetworkPolicy, and delegates Pod IP management and network connectivity to the
+primary CNI.
+
+## Design
+
+Antrea is designed to work as NetworkPolicy plug-in for the routed CNIs. For as long as a CNI
+implementation fits into this model, Antrea may be inserted to enforce NetworkPolicy in that
+CNI's environment.
+
+<img src="/docs/assets/policy-only-cni.svg" width="600" alt="Antrea Switched CNI">
+
+The above diagram depicts a routed CNI network topology on the left, and what it looks like 
+after Antrea inserts the OVS bridge into the data path.
```

Comment:
It looks unexpected to see "OVS bridge" suddenly here which is never mentioned before. Consider either changing to "after Antrea is inserted and enforces NetworkPolicies with OVS", or introducing OVS earlier, like in line 10 (e.g. "to enforce NetworkPolicy with OVS").